Makefile

```diff
@@ -366,6 +373,46 @@ $(COMPLETED)/uthash: vendor/uthash/.success_fetching_source $(COMPLETED)/builddi
 	touch $(COMPLETED)/uthash
 uthash: $(COMPLETED)/uthash
 
+vendor/libaio/.success_retrieving_source:
+	rm -rdf vendor/libaio/
+	mkdir vendor/libaio
+	wget -O- http://ftp.de.debian.org/debian/pool/main/liba/libaio/libaio_$(LIBAIOVERSION).orig.tar.xz | tar xJ -C vendor/libaio -f -
+	mv vendor/libaio/libaio-$(LIBAIOVERSION)/* vendor/libaio
+	rm -rdf vendor/libaio/libaio-$(LIBAIOVERSION)
```

Comment:
This directory *should* be empty at this point, right?  We can `rmdir` here instead of `rm -rdf`.  The advantage of doing this is that, if somehow our expectation is wrong, we'll get a well placed error message, which can save on debugging efforts.

Reply:
I think there is a .gitignore in the tarball and because of that it failed. Should I copy dotfiles or delete them?

Reply:
Ahh, okay.  That does explain it.  In that case I the `rm -r` is fine.  Apologies, I probably should have checked instead of assuming here.

Given the discussion about using git we're having elsewhere, this'll likely be a non-issue either way.